### PR TITLE
Update mapping in csv-import-mapping.json

### PR DESCRIPTION
**Pull request by souravcipher**
This PR includes the latest mapping

### CODE REVIEW

File: env.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ debug.enable(process.env.DEBUG);
 
 const defaults = {
   NODE_ENV: 'development',
-  API_KEY: '09u624Pc9F47zoGLlkg1TBSbOl2ydSAq',
```
Comment (souravcipher) [inline env.js:21]: This is an old key and already deleted!

Comment (znarf) [reply env.js:21]: You should not touch that file.

File: env.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ debug.enable(process.env.DEBUG);
...
-  API_URL: 'https://api-staging.opencollective.com',
```
Comment (znarf) [inline env.js:22]: You should not touch that file.

**Comment by znarf** [comment]
I updated the instructions at https://github.com/opencollective/opencollective-tools#prerequisite

`PERSONAL_TOKEN` (renamed from API_KEY) and `API_URL` should be put in a file named `.env`

`env.js` should never be touched.

**Comment by souravcipher** [comment]
> I updated the instructions at https://github.com/opencollective/opencollective-tools#prerequisite
> 
> `PERSONAL_TOKEN` (renamed from API_KEY) and `API_URL` should be put in a file named `.env`
> 
> `env.js` should never be touched.

Thank you so much @znarf, I have updated the PR!!

**Review by znarf — APPROVED** [review]

**Pull request merged**